core/lib/Thelia/Core/Template/Loop/Order.php

```diff
@@ -320,13 +321,21 @@ public function buildModelCriteria()
      */
     public function parseResults(LoopResult $loopResult)
     {
+        $lastLegacyOrderId = ConfigQuery::read('last_legacy_order_id', 0);
```

Comment:
The term legacy is not really clear maybe last_miscalculated_order_id or last_not_rounded_order_id ?

Reply:
`last_old_style_rounding_order_id` ?